src/tests/unit/findWithPagination.test.ts

```diff
@@ -48,39 +49,64 @@ afterEach(async () => {
   await connection.query("ROLLBACK").catch(() => {});
   jest.resetAllMocks();
 });
-
 describe("pagination", () => {
   beforeAll(async () => {
     const personRepository = connection.getRepository(Person);
     const ownerA = await personRepository.save({ name: "A" });
     const ownerB = await personRepository.save({ name: "B" });
 
+
     await connection.getRepository(Foo).insert([
-      { foo: "page-test-a", owner: ownerA },
-      { foo: "page-test-b", owner: ownerB },
-      { foo: "page-test-c", owner: ownerA },
+      { foo: "page-test-a", owner: ownerA, id: 'a' },
```

Comment:
I hard-coded the ids here instead of relying on an auto incremented uuid to ensure the assertion one line 94 is possible. Without knowing the id order of each Foo, you cannot assert that the ordering worked correctly.